src/test/scala/mesosphere/mesos/TaskBuilderTest.scala

```diff
@@ -24,11 +24,7 @@ class TaskBuilderTest extends MarathonSpec {
   import mesosphere.mesos.protos.Implicits._
 
   test("BuildIfMatches") {
-    val offer = makeBasicOffer(cpus = 1.0, mem = 128.0, disk = 2000.0, beginPort = 31000, endPort = 32000)
-      .addResources(ScalarResource("cpus", 1))
-      .addResources(ScalarResource("mem", 128))
-      .addResources(ScalarResource("disk", 2000))
-      .build
+    val offer = makeBasicOffer(cpus = 1.0, mem = 128.0, disk = 2000.0, beginPort = 31000, endPort = 32000).build
```

Comment:
Yes it is, I cleaned up the rest of the file and put those changes in a separate commit.